templates.tf

```diff
@@ -14,7 +14,7 @@ data "template_file" "ranger_admin" {
   {
     "name": "ranger-admin",
     "image": "${var.ranger_docker_image}:${var.ranger_docker_version}",
-    "${local.docker_auth}"
+    ${local.docker_auth}
     "command": [ "/start-ranger-admin.sh" ],
```

Comment:
Without this change, I am getting `Errors: [ECS Task Definition container_definitions is invalid: Error decoding JSON: invalid character '"' after object key]` whenever I try to deploy this module.  It looks like this same fix has already been made in the datalake repo: https://github.com/ExpediaGroup/apiary-data-lake/blob/master/templates/apiary-hms-readwrite.json#L5